middleware/grant-attacher.js

```diff
@@ -15,8 +15,12 @@
  */
 'use strict';
 
-module.exports = function (keycloak) {
+module.exports = function (keycloakAdapters) {
   return function grantAttacher (request, response, next) {
+    const keycloak = request.session.realmInfo && request.session.realmInfo.name ?
+      keycloakAdapters[request.session.realmInfo.name] :
+      keycloakAdapters['Default-Realm'];
```

Comment:
Is there anywhere reasonable that we can centralize this kind of logic?

Reply:
Centralized the logic in a function exported from a utils file.